spec/reports/rare_uncommitted_spec.rb

```diff
@@ -277,4 +277,32 @@ def report(kwargs)
       expect(counts).to eq expected
     end
   end
+
+  describe "#output_organization" do
+    it "returns an array of report rows, incl header" do
+      cluster_tap_save [hti1, hol1_org1, hti2, hol2_org1, spc2_org1]
+      out_arr = report(organization: org1, max_h: 5).output_organization.to_a
+      expect(out_arr).to be_a Array
+      expect(out_arr.size).to eq 2
+      compare_header = [
+        "member_id",
+        "local_id",
+        "gov_doc",
+        "condition",
+        "OCN",
+        "overlap_ht",
+        "overlap_sp"
+      ].join("\t")
+      header = out_arr.shift
+      expect(header).to eq compare_header
+
+      row_data = out_arr.shift.split("\t")
+      expect(row_data[0]).to eq "umich"
+      expect(row_data[1]).to eq hol1_org1.local_id
+      expect(row_data[2]).to eq hol1_org1.gov_doc_flag ? "1" : "0"
```

Comment:
Duplicates logic in the code, but seems hard to avoid unless we set `gov_doc_flag` to a known value.

Reply:
Well if e.g. `row_data[1]` _didn't equal_ `hol1_org1.local_id` we sure would want to know.